backend/src/test/java/de/neuefische/backend/controller/APIControllerTest.java

```diff
@@ -0,0 +1,7 @@
+package de.neuefische.backend.controller;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class APIControllerTest {
+
```

Comment:
I think there is something missing here

Reply:
only prepared the test class. still have to write the tests